pyroute/utils.py

```diff
@@ -85,30 +133,41 @@ def get_shared_attrs(objs):
     @staticmethod
     def get_methods(objs):
         """
-        This method takes a dictionary of objects of the form:
-            - {object_name: object_instance}
-        and returns their methods, in a dictionary of the form:
-            - {method_name: object_instance}
+        This method takes a dictionary of objects and returns their
+        methods in a dictionary.
+
         If two or more objects share the same method, only one will
-        be loaded.        
+        be loaded.
+
+        :param objs: The objects to be processed
+        :type objs: Dictionary
+        :returns shared: Dictionary
         """
-        return {method: obj for obj in objs.values() 
+        return {method: obj for obj in objs.values()
                 for method in Utils.clear_dunders(dir(obj))
                 if callable(getattr(obj, method))}
 
     @staticmethod
     def clean_filenames(filenames):
         """
         This method takes a list of filenames and gets rid of their
-        extensions
+        extensions.
+
+        :param filenames: A List of filenames to be processed_func
+        :type filenames: List <String>
+        :returns set: Set of Methods
         """
         return set(map(lambda x: os.path.splitext(x)[0], filenames))
 
     @staticmethod
     def clear_dunders(methods):
         """
-        This method takes a list of methods and filters out those 
-        that are Python's 'dunder' methods
+        This method takes a list of methods and filters out those
+        that are Python's 'dunder' methods.
+
+        :param methods: A list of Methods and Filters
+        :type methods: List <String>
+        :returns List: A list of dunder methods
```

Comment:
I think you mean "A list of non-dunder methods" here, as this method deletes all dunder methods.